system/kube-monitoring/charts/prometheus-frontend/openstack-swift.alerts

```diff
@@ -0,0 +1,184 @@
+ALERT SwiftAccountFirstByteTiming
+  IF max(swift_proxy_firstbyte_timer{status="200",type="account",quantile="0.5"}) by (cluster) > 500.0
+  FOR 3m
+  LABELS {
+    service = "swift"
+    severity = "critical"
+    context = "openstack"
+    dashboard = "swift-overview"
```

Comment:
`swift-proxy` dashboard. There are vars for cluster and proxy instance for preselection available like ```swift-proxy?var-cluster={{`{{$labels.os_cluster}}`}}&var-proxy={{`{{$labels.instance}}`}}```